<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SshAdvanced</class>
 <widget class="QWidget" name="SshAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>290</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chk_useCustomGatewayPort">
       <property name="text">
        <string>Use custom gateway port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_useCustomGatewayPort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chk_useCustomTunnelMtu">
       <property name="text">
        <string>Use custom tunnel Maximum Transmission Unit (MTU):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_useCustomTunnelMtu">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="chk_remoteDeviceNumber">
       <property name="text">
        <string>Remote device number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_remoteDeviceNumber">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chk_useTapDevice">
     <property name="text">
      <string>Use a TAP device</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="chk_remoteUsername">
       <property name="text">
        <string>Remote username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_remoteUsername">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="chk_noTunnelInterface">
       <property name="text">
        <string>No tunnel (specify dummy interface):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_noTunnelInterface">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="chk_socksBindAddress">
       <property name="text">
        <string>SOCKS bind addresses (-D, space separated):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_socksBindAddress">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="chk_localBindAddress">
       <property name="text">
        <string>Local bind addresses (-L, space separated):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_localBindAddress">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="chk_remoteBindAddress">
       <property name="text">
        <string>Remote bind addresses (-L, space separated):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_remoteBindAddress">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chk_useCustomGatewayPort</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_useCustomGatewayPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_useCustomTunnelMtu</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_useCustomTunnelMtu</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_remoteDeviceNumber</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_remoteDeviceNumber</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_remoteUsername</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_remoteUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_noTunnelInterface</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_noTunnelInterface</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_socksBindAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_socksBindAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_localBindAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_localBindAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_remoteBindAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_remoteBindAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
